Vagrantfile

```diff
@@ -97,6 +97,11 @@ if File.exist?(env_specific_config_path)
   end
 end
 
+# Display the main current configuration parameters
+@ui.info "Welcome to Kubernetes playground !"
+@ui.info "Vagrant provider : " + settings["conf"]["vagrant_provider"]
+@ui.info "Networking plugin : " + settings["ansible"]["group_vars"]["all"]["kubernetes_network_plugin"]
```

Comment:
1) `Currently selected Kubernetes networking plugin:`
2) Same thing about the variable. Did we define one?

Reply:
see above

Reply:
BTW in this case we do not have a variable... so it would be case by case...

Reply:
Ok, let's not lose time on this one